src/layout/section/columnedContentSection/index.tsx

```diff
@@ -9,7 +9,7 @@ const StyledColumnedContentSection = styled(ColumnedContentSection)`
   }
 
   & .kirk-columned-content-section-title {
-    margin: ${space.l} 0;
+    margin: ${space.xl} 0;
```

Comment:
Don't we want to use padding instead? Since margin are merged if the elements after or before also have margins, it's probably working in this case though so not blocker

Reply:
yep, swapped to padz.